src/replicate_stage.rs

```diff
@@ -42,7 +42,15 @@ impl ReplicateStage {
         }
         let entries = reconstruct_entries_from_blobs(blobs.clone())?;
 
-        let res = bank.process_entries(entries.clone());
+        let mut stats = BankErrorStats::default();
+
+        let res = bank.process_entries(entries.clone(), &mut stats);
```

Comment:
actually it doesn't really work because `process_entries` will have an early exit when processing errors so the count will not be correct. Seems like the bank kind of has to have that leader information.

Reply:
I don't follow. ReplicateStage is only in the TVU.